src/Helpers/helpers.php

```diff
@@ -109,3 +109,28 @@ function money_to_decimal(Money $money)
         return (new DecimalMoneyFormatter(new ISOCurrencies()))->format($money);
     }
 }
+
+if (! function_exists('rounded_type')) {
```

Comment:
I think this should be in a separate (action) class or a method on `FirstPaymentSubscriptionBuilder`